community/people/emd/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: person
+title: Enrique Miralles-Dolz
+image: emd.jpg
+role: Associate Research Engineer, Princeton Plasma Physics Laboratory
```

Comment:
```suggestion
role: Associate Research Engineer, (visiting from) Princeton Plasma Physics Laboratory
```

Reply:
Yep I agree with this. I wasn't sure about whether I should specify that I am a visiting researcher. Maybe I can add 'Visiting Researcher' in people.yml and my PPPL role in index.md?

Reply:
This way when one lands on the Community/people site it will read 'Visiting Researcher', and if they go to my profile it will display my role at PPPL.